### PR TITLE
fix: updating color pallette

### PR DESCRIPTION
**Pull request by smyrn96**
## Description

This PR is created to update the color pallete we use in Plus.

## Changes

- Dark and darker theme colors removed.
- The colors of the components have been updated.

### CODE REVIEW

**Review by SteliosKon — APPROVED** [review]

File: src/theme/default/colors.ts [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import Color from "color";
 
+//updating color pallette
```
[the remark this answers is not in the document]
Comment (smyrn96) [reply src/theme/default/colors.ts]: Fixed: [ada76ba](https://github.com/epignosis/gnosis/pull/145/commits/ada76ba0078763616de5cb14f2be729e9001c697)

File: src/theme/default/config/formElements.ts [hunk 2]
```diff
@@ -6,36 +6,41 @@ export default {
...
+      shadowColor: Color(colors.primary.lightest).alpha(0.25).string(),
+    },
+    disabled: {
+      borderColor: colors.secondary.base,
     },
   },
   generalError: {
     borderColor: colors.red.base,
-    background: colors.red.lightest,
+    background: Color(colors.red.light).alpha(0.15).string(),
```
Comment (ThrasyvoulosKafasis) [inline src/theme/default/config/formElements.ts:44]: src/components/FormElements/FormError/styles.ts line 8 remove hexToRGBA

**Review by ThrasyvoulosKafasis — CHANGES_REQUESTED** [review]

File: src/theme/default/config/formElements.ts (hunk 2 above)
Comment (smyrn96) [reply src/theme/default/config/formElements.ts:44]: Fixed: [fd23ed7](https://github.com/epignosis/gnosis/pull/145/commits/fd23ed764f482768ab0c66a145e14314bddf64a5)

File: src/components/Table/styles.ts [hunk 3]
```diff
@@ -14,7 +14,7 @@ export const tableContainer = ({ table }: Theme) => css`
     max-width: 100%;
 
     thead {
-      border-bottom: 1px solid ${hexToRGBA(colors.gray.lightest.toString(), 0.4)};
+      border-bottom: 1px solid ${hexToRGBA(colors.secondary.lightest.toString(), 0.4)};
```
Comment (ThrasyvoulosKafasis) [inline src/components/Table/styles.ts:17]: Create a headerBorder prop in src/theme/default/config/table.ts and replace the border bottom color

**Review by ThrasyvoulosKafasis — CHANGES_REQUESTED** [review]

File: src/components/Table/styles.ts (hunk 3 above)
Comment (smyrn96) [reply src/components/Table/styles.ts:17]: Fixed: [b1a3710](https://github.com/epignosis/gnosis/pull/145/commits/b1a37108b1a6dc906d78f4a47226c38dce79b78b)

File: src/theme/default/config/formElements.ts [hunk 4]
```diff
@@ -6,36 +6,41 @@ export default {
...
+    inputBorderColor: colors.secondary.lighter,
+    background: colors.secondary.lighter,
     backgroundFocus: colors.white,
     iconColor: colors.black,
-    hoverColor: "#F8F8F8",
-    disabledBackground: "#eeeeee",
-    disabledBorder: "#cccccc",
-    disabledIcon: "#C9C9C9",
+    hoverColor: colors.secondary.lighter,
+    disabledBackground: colors.secondary.lighter,
```
[the remark this answers is not in the document]
Comment (smyrn96) [reply src/theme/default/config/formElements.ts:16]: Fixed: [b1a3710](https://github.com/epignosis/gnosis/pull/145/commits/b1a37108b1a6dc906d78f4a47226c38dce79b78b)

File: src/components/Table/styles.ts (hunk 3 above)
Comment (smyrn96) [reply src/components/Table/styles.ts]: Fixed: [0bfc9b9](https://github.com/epignosis/gnosis/pull/145/commits/0bfc9b9fbdf2f6f2929be2d5d943e3634ad2ff91)

**Review by ThrasyvoulosKafasis — APPROVED** [review]

**Pull request merged**